README.md

```diff
@@ -14,6 +14,8 @@ This crate is inspired by java  [StrictMath](https://github.com/openjdk/jdk/blob
 - sinh
 - tan
 - tanh
+- log
```

Comment:
The function list is sorted in alphabetical order,  please move the **log** function after **hypot**.